static/lang/en-GB.json

```diff
@@ -221,6 +221,7 @@
                 "image_width": "Maximum width of images (percent)",
                 "preview_info": "These options determine which elements are rendered in documents.",
                 "render_citations": "Render Citations",
+                "render_codeblock_button": "Render codeblock copy button",
```

Comment:
Do not alter the language files. These depend on the translation services, so all your changes would be overridden the next time the CI refreshes the localizations. Instead, please provide a list of all newly added translation strings together with a suggested English wording in your PR description. This way I can easily copy those to Translate and they will be picked up by the CI.

Reply:
Sorry, I am not really understand. Currently, I just remove that line in our repo